lapce-proxy/src/plugin/mod.rs

```diff
@@ -806,6 +806,9 @@ pub fn download_volt(volt: VoltInfo, wasm: bool) -> Result<VoltMetadata> {
         let url = url.join(wasm)?;
         {
             let mut resp = reqwest::blocking::get(url)?;
+            if let Some(path) = path.join(&wasm).parent() {
+                fs::DirBuilder::new().recursive(true).create(path)?;
```

Comment:
Maybe check if the path exits first?

Reply:
> It is considered an error if the directory already exists unless recursive mode is enabled.

Reply:
https://doc.rust-lang.org/std/fs/struct.DirBuilder.html#method.create

Reply:
It feels unnecessary if it exists even if it doesn’t throw an error.